UKHomeOffice/sgmr-service
# SPCR-145 Rename “Submit a pleasure craft report” to “Tell Border Force and HMRC you are sailing to or from the UK in a pleasure craft”

## Description
We have new user flows being tested with users. In the meantime, we should take the content on the prototype and update the existing pages on the service to match.

All user-visible mentions of "Submit a Pleasure Craft Report" have been replaced by “Tell Border Force and HMRC you are sailing to or from the UK in a pleasure craft”.

## To Test
- Go to any page which would have previously contained "Submit a Pleasure Craft Report"
- Ctrl + F in your browser and type "Submit a Pleasure Craft Report"
- You should get no results
- Ctrl + F in your browser and type “Tell Border Force and HMRC you are sailing to or from the UK in a pleasure craft”
- You should get at least one result

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
